<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="sushants-MacBook-Air.local" name="com.qa.furniture.test.FurniturePutTest" tests="1" failures="0" timestamp="16 Feb 2020 06:45:52 GMT" time="0.623" errors="1">
  <testcase name="putTest" time="0.623" classname="com.qa.furniture.test.FurniturePutTest">
    <error type="org.testng.TestNGException" message="
The data provider is trying to pass 10 parameters but the method com.qa.furniture.test.FurniturePutTest#putTest takes 1">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 10 parameters but the method com.qa.furniture.test.FurniturePutTest#putTest takes 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1257)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1151)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- putTest -->
</testsuite> <!-- com.qa.furniture.test.FurniturePutTest -->
